apps/hefty/lib/utils/datatime.ex

```diff
@@ -17,33 +17,18 @@ defmodule Hefty.Utils.Datetime do
     [from_timestamp * 1000, to_timestamp * 1000]
   end
 
-  def get_last_day(datetime) do
-    to_datetime = NaiveDateTime.from_iso8601!("#{datetime}")
-
-    to_timestamp =
-      to_datetime
-      |> DateTime.from_naive!("Etc/UTC")
-      |> DateTime.to_unix()
-
-    from_datetime = NaiveDateTime.add(to_datetime, -86400, :second)
-
-    from_timestamp =
-      from_datetime
-      |> DateTime.from_naive!("Etc/UTC")
-      |> DateTime.to_unix()
-
-    [from_timestamp * 1000, to_timestamp * 1000]
-  end
+  def get_last(n, interval, datetime) do
+    times = %{:day => 86_400, :week => 604_800, :year => 31_536_000}
```

Comment:
`times` is not the best name, `seconds` or `seconds_map`

Reply:
I was changed to `seconds_map`